jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/DeploymentManager.java

```diff
@@ -242,264 +211,130 @@ public void insertLifeCycleNode(String existingFromNodeName, String existingToNo
         _lifecycle.insertNode(edge, insertedNodeName);
     }
 
-    @Override
-    protected void doStart() throws Exception
+    public boolean isUseStandardBindings()
     {
-        if (getContexts() == null)
-            throw new IllegalStateException("No Contexts");
-
-        if (_useStandardBindings)
-        {
-            LOG.debug("DeploymentManager using standard bindings");
-            addLifeCycleBinding(new StandardDeployer());
-            addLifeCycleBinding(new StandardStarter());
-            addLifeCycleBinding(new StandardStopper());
-            addLifeCycleBinding(new StandardUndeployer());
-        }
-
-        // Start all of the AppProviders
-        for (AppProvider provider : _providers)
-        {
-            startAppProvider(provider);
-        }
```

Comment:
I think by not having a relationship to the app providers here, we have changed the start order.  Now the log lines like:
```
2025-03-03 18:43:08.807:INFO :oejs.Server:main: Started oejs.Server@7e5d9a50{STARTING}[12.1.0-SNAPSHOT,sto=0] @1621ms
```

are coming out before the deployer has deployed any webapps, so at the very least, the time reported is not correct and any tests/environments scanning for the "started" log will act too soon before any applications have been deployed.

Ir also means that the order of starting connectors after deploying apps has changed.  We cannot change this order.    So can you find another way to retain that order?